mosaic-site/src/components/card/Card.tsx

```diff
@@ -34,7 +36,7 @@ export const Card = ({
   keylineColor,
   keyLineAnimation = true,
 }: CardProps): JSX.Element => {
-  const ref = useRef<HTMLDivElement>();
+  const ref = useRef<HTMLDivElement>() as MutableRefObject<HTMLDivElement>;
```

Comment:
would suggest same as below but useOnScreen needs non null, maybe worth seeing if we can change the types in that hook in mosaic